packages/core/src/browser/quick-input/quick-command-service.ts

```diff
@@ -42,12 +45,93 @@ export class QuickCommandService extends QuickInputService implements QuickAcces
     @inject(CorePreferences)
     protected readonly corePreferences: CorePreferences;
 
+    @inject(QuickAccessRegistry)
+    protected readonly quickAccessRegistry: QuickAccessRegistry;
+
+    constructor(@inject(KeybindingRegistry) protected readonly keybindingRegistry: KeybindingRegistry) {
+    }
```

Comment:
do we really need an empty constructor?
can we use [the property injection](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#dependency-injection) here instead?

Reply:
No we don't.